### PR TITLE
Address mnot's review (modulo problem types)

### PR DESCRIPTION
**Pull request by chris-wood**
See [the review here](https://mailarchive.ietf.org/arch/msg/privacy-pass/UY60WAavqC-ailiT2wgMINFfloQ/). This does not address the problem type question, which we should take to the list. 

cc @mnot in case you have spare eyes and time during TPAC.

### CODE REVIEW

File: draft-ietf-privacypass-protocol.md [hunk 1]
```diff
@@ -332,15 +332,12 @@ The values `token_input` and `blinded_element` are stored locally and used
...
+"application/private-token-request". An example request for the Issuer Request URL
+"https://issuer.example.net/request" is shown below.
 
 ~~~
-:method = POST
-:scheme = https
-:authority = issuer.example.net
-:path = /request
+POST /request HTTP/1.1
 accept = application/private-token-response
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:340]: These field names should be upper-case for H1 examples

File: draft-ietf-privacypass-protocol.md [hunk 2]
```diff
@@ -332,15 +332,12 @@ The values `token_input` and `blinded_element` are stored locally and used
 later as described in {{private-finalize}}. The Client then generates an HTTP
 POST request to send to the Issuer Request URL, with the TokenRequest as the
 content. The media type for this request is
-"application/private-token-request". An example request is shown below.
+"application/private-token-request". An example request for the Issuer Request URL
+"https://issuer.example.net/request" is shown below.
 
 ~~~
-:method = POST
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:338]: I think you also need a Host header here

Comment (chris-wood) [reply draft-ietf-privacypass-protocol.md:338]: Fixed

**Review by tfpauly — APPROVED** [review]

**Comment by mnot** [comment]
Took a quick look, lgtm - thanks!

**Pull request merged**